helpers/build_packages.sh

```diff
@@ -151,7 +151,60 @@ if [ "$BUILDMW" = "1" ]; then
 
     pushd $ANDROID_ROOT/hybris/mw > /dev/null
 
-    if [ "$BUILDMW_REPO" = "" ]; then
+    MANIFEST_LOOKUP=$ANDROID_ROOT
```

Comment:
Paths like this should be quoted or they brake when they contain spaces.

Reply:
That's true for the cases below, but in this simple assignment case it should work without quotes, shellcheck doesn't complain about that (or could you show a case where it breaks?)

```bash
mkdir "space space"
spc=$(ls -d space*)
echo $spc
assigning=$spc
echo $assigning
```

Reply:
Ah yes you're right. Shell quoting rules are weird sometimes.